homeassistant/components/flux_led/light.py

```diff
@@ -225,6 +224,15 @@ async def async_new_lights(bulbs: dict):
 
     async_dispatcher_connect(hass, SIGNAL_ADD_DEVICE, async_new_lights)
 
+    # register custom_effect service
+    platform = entity_platform.current_platform.get()
+
+    platform.async_register_entity_service(
+        SERVICE_CUSTOM_EFFECT,
```

Comment:
Didn't we already merge the effect service?

Reply:
I thought you did.

Reply:
Yes, it's merged so please rebase on latest fetch of the target branch:
https://github.com/home-assistant/core/tree/rework-flux_led

Reply:
Rebase done. Thanks - forgot to do that!